.github/workflows/pull_request.yml

```diff
@@ -34,13 +39,16 @@ jobs:
           branch_name='preview-${{ github.event.pull_request.number }}'
           echo branch doesnt exist
           git checkout -b $branch_name || echo branch exists
-          git push --set-upstream origin $branch_name
+          git push --set-upstream origin $branch_name || echo echo branch exists remotely
```

Comment:
`echo echo` doesn't look right.

Reply:
Oops. Yeah not sure how that got in there. 
Good catch